distributed/tests/test_client.py

```diff
@@ -3724,6 +3725,11 @@ def test_get_versions(c):
     assert dict(v["client"]["packages"]["optional"])["requests"] == requests.__version__
 
 
+@pytest.mark.asyncio
+async def test_async_get_versions(ac):
```

Comment:
Currently we use the `gen_cluster` decorator for this:

```suggestion
@gen_cluster(client=True)
async def test_async_get_versions(c, s, a, b):
```

This does a bunch of other things that are useful.  Here are a few:

1.  It's much faster than using `LocalCluster`, probably something like 10ms rather than a couple seconds
2. It gives us some workers and the scheduler too
3.  It does a ton of checks on cleanup (see the cleanup fixture)
4.  It adds in the `pytest.mark.asyncio` decorator by default (you should also remove that line)

Reply:
Ah nice!